### PR TITLE
Add Radiomaster Ranger targets

### PR DESCRIPTION
**Pull request by pkendall64**
Adds support for the following targets
- Radiomaster Ranger
- Radiomaster Ranger Nano
- Radiomaster Ranger Micro
- Radiomaster ER5 A/C PWM reveivers

### CODE REVIEW

**Review by FOG-Yamato — APPROVED** [review]

File: src/targets/radiomaster_2400.ini [hunk 1]
```diff
@@ -21,3 +21,24 @@ extends = env:RadioMaster_TX16S_2400_TX_via_ETX
 
 [env:RadioMaster_TX16S_2400_TX_via_WIFI]
 extends = env:RadioMaster_TX16S_2400_TX_via_ETX
+
+[env:RadioMaster_Ranger_2400_TX_via_UART]
+extends = env:Unified_ESP32_2400_TX_via_UART
+board_config = radiomaster.tx_2400.ranger
+
+[env:RadioMaster_Ranger_2400_TX_via_WIFI]
+extends = env:RadioMaster_Ranger_2400_TX_via_ETX
```
Comment (jurgelenas) [inline src/targets/radiomaster_2400.ini:30]: `RadioMaster_Ranger_2400_TX_via_ETX` is incorrect. Should be `RadioMaster_Ranger_2400_TX_via_UART`.

**Review by jurgelenas — DISMISSED** [review]
Need to fix Ranger Wifi target.

File: src/targets/radiomaster_2400.ini (hunk 1 above)
Comment (pkendall64) [reply src/targets/radiomaster_2400.ini]: Good catch, I completely missed that one. I might have to extend the validator 😉

File: src/hardware/targets.json [hunk 2]
```diff
@@ -1211,9 +1211,45 @@
...
+                "overlay": {
+                    "power_values": [-18,-15,-12,-8,-5,0]
+                },
+                "upload_methods": ["uart", "wifi"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
             }
         },
         "rx_2400": {
+            "er5": {
```
Comment (jurgelenas) [inline src/hardware/targets.json:1244]: No platformio target?

Comment (pkendall64) [reply src/hardware/targets.json:1244]: No we don't need one. Configurator can use the alias, and I'm just adding it to targets for completeness for when configurator can use it 😄

Comment (jurgelenas) [reply src/hardware/targets.json:1244]: Ok. Perfect!

File: src/hardware/targets.json [hunk 3]
```diff
@@ -1211,9 +1211,45 @@
...
+                "lua_name": "RadioMstr Ranger",
+                "layout_file": "Radiomaster Ranger.json",
+                "upload_methods": ["uart", "wifi"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
+            },
+            "ranger-micro": {
+                "product_name": "RadioMaster Ranger Micro 2.4GHz TX",
+                "lua_name": "RM Ranger Micro",
+                "layout_file": "Ranger Micro.json",
```
Comment (JyeSmith) [inline src/hardware/targets.json:1226]: `Radiomaster Ranger Micro.json`

Needs Radiomaster added to the file name.

File: src/hardware/targets.json [hunk 4]
```diff
@@ -1211,9 +1211,45 @@
...
+                "upload_methods": ["uart", "wifi"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
+            },
+            "ranger-nano": {
+                "product_name": "RadioMaster Ranger Nano 2.4GHz TX",
+                "lua_name": "RM Ranger Nano",
+                "layout_file": "Ranger Micro.json",
```
Comment (JyeSmith) [inline src/hardware/targets.json:1234]: Update file name.

**Review by JyeSmith — APPROVED** [review]

**Pull request merged**